modules/ppcp-compat/services.php

```diff
@@ -49,6 +49,7 @@
 			'ppcp-gateway-settings',
 			'ppcp-webhooks-status-page',
 			'ppcp-tracking',
+			'ppcp-fraudnet',
```

Comment:
Seems like it's also missing `ppcp-gzd-compat`, `ppcp-clear-db`. (found via Ctrl-Shift-F regex mode `wp_(e|r).+_script\(`).